front-end/src/UserHomePage.js

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+import TopNavigation from "@cloudscape-design/components/top-navigation";
+import Container from "@cloudscape-design/components/container";
+import Header from "@cloudscape-design/components/header";
+import ContentLayout from "@cloudscape-design/components/content-layout";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+import Button from "@cloudscape-design/components/button";
+import Badge from "@cloudscape-design/components/badge";
+
+export default function UserHomePage (){
+  const [data, setData] = React.useState([
+    { title:"Employee survey 1", due: ('2023-10-20'),id:1},
+    { title:"Employee survey 2", due: ('2023-09-30'),id:2},
+    { title:"Employee survey 3", due: ('2023-09-12'),id:3}
+  ]);
+
+  const today = new Date();
+  const month = today.getMonth()+1;
+  const year = today.getFullYear();
+  const date = today. getDate();
+  const currentDate = year + "/" + month + "/" + date;
+  console.log(currentDate);
+
+  return (
+    <React.Fragment>
+    <TopNavigation
+      identity={{
+        href: "#",
+        title: "Internal Survey Tool",
+        logo: {
+          src:"https://cdn4.vectorstock.com/i/1000x1000/88/53/document-with-stamp-and-pen-business-documents-vector-31048853.jpg",
```

Comment:
Download the image and use the local path.

Reply:
Understood